src/GraphicsDrawer.cpp

```diff
@@ -1233,7 +1233,7 @@ void GraphicsDrawer::drawTexturedRect(const TexturedRectParams & _params)
 	TextureCache & cache = textureCache();
 	const bool bUseBilinear = gDP.otherMode.textureFilter != 0;
 	const bool bUseTexrectDrawer = m_bBGMode || ((config.graphics2D.enableNativeResTexrects != 0)
-		&& bUseBilinear
+		//	&& bUseBilinear
```

Comment:
Why? Native res texrects were invented to fix issues with bilinear filtering.
Unfiltered texrects usually worked fine. 
What is changed?

Reply:
The problem is a conflict with this. Scale is computed equally for all texrects.
https://github.com/gonetz/GLideN64/blob/c8a25914bcc47a176aad12cb5e9c2ffdc3ff3847/src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramUniformFactory.cpp#L245-L249
It is also possible to change this condition to `config.graphics2D.enableNativeResTexrects != 0 && gDP.otherMode.textureFilter = Texture`.

> Unfiltered texrects usually worked fine.

The Ogre Battle case is one where unfiltered rects don't render correctly. It is possible to render correctly in native, but not in higher resolutions, due to dsdx being too big. Unfortunately, the scene is not correct with the texrect drawer enabled either, not sure why.

> What is changed?

Mario Kart press start menu with enableNativeResTexrects enabled has an issue after the latest changes, where an incorrect texture coordinate bias is being applied.